### PR TITLE
Document testing

### PR DESCRIPTION
**Pull request by ubergesundheit**
This PR adds information about how this chart is being tested.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,3 +5,17 @@
...
 -->
+
+This PR...
+
+## Checklist
+
+- [ ] Automated test are working
+- [ ] Changelog entry has been added
+
+### Manual tests on workload clusters (not always required)
```
Comment (JosephSalisbury) [inline .github/pull_request_template.md:16]: Can we change to something like, required on chart changes etc. i.e - when would they be required?

File: README.md [hunk 2]
```diff
@@ -162,6 +162,18 @@ This is not required for Helm 3, as CRDs will be installed automatically.
 
 * https://github.com/Kong/charts/
 
+## Automatic and manual testing
+
+The helm chart in this repository undergoes [a series of automated tests](tests/ats/test_basic_cluster.py) running on a [kind](https://kind.sigs.k8s.io/) cluster ([kind cluster config](tests/kind_config.yaml)) executed by [app-test-suite](https://github.com/giantswarm/app-test-suite). ([chart values used for tests](tests/test-values.yaml))
+
+Testing includes creation of a `Deployment`, `Service` and `Ingress` resources to check if reconciliation works as intented.
```
Comment (JosephSalisbury) [inline README.md:169]:
```suggestion
Testing includes creation of`Deployment`, `Service`, and `Ingress` resources to check if reconciliation works as intended.
```

**Review by webwurst — APPROVED** [review]
Sounds good. I'll try the steps and add details if useful.

File: .github/pull_request_template.md (hunk 1 above)
Comment (ubergesundheit) [reply .github/pull_request_template.md:16]: yes

File: .ats/main.yaml [hunk 3]
```diff
@@ -10,3 +10,7 @@ upgrade-tests-cluster-type: kind
 upgrade-tests-cluster-config-file: tests/kind_config.yaml
 upgrade-tests-app-catalog-url: https://giantswarm.github.io/giantswarm-catalog
 upgrade-tests-app-config-file: tests/test-values.yaml
+
+# use this configuration iption to test upgrade
```
Comment (webwurst) [inline .ats/main.yaml:14]: > iption
? ;D

**Comment by ubergesundheit** [comment]
Ok waiting on input from @webwurst before merge

**Review by webwurst — APPROVED** [review]
okidoki

**Comment by ubergesundheit** [comment]
I assume you did not found anything, otherwise just open a separate PR :)

**Pull request merged**